graylog2-server/src/main/java/org/graylog2/streams/StreamRouterEngine.java

```diff
@@ -180,7 +216,7 @@ public String getFingerprint() {
 
             for (final StreamRule streamRule : stream.getStreamRules()) {
                 try {
-                    final Rule rule = new Rule(stream, streamRule);
+                    final Rule rule = new Rule(stream, streamRule, null);
```

Comment:
It is not relevant for the reuse of this class in the scope of the test matches, as shortcuts are not taken here and all rules need to be evaluated.